daml-script/test/daml/upgrades/UpgradesTest.daml

```diff
@@ -53,6 +55,21 @@ main = forA_ tests $ \(testName, test) -> do
   debugRaw $ "Testing: " <> testName
   test
 
+-- | Used to tag a test as failing by erroring in any way, once all this behaviour works, this function can be removed
+failingScript : Script () -> Script ()
+failingScript act = do
+  tryToEither (\() -> liftFailedCommandToException act) >>= \case
+    Right _ -> assertFail "Expected failed and got success! Did you fix this logic? Remove the wrapping `failure` to mark this as working."
+    Left _ -> do
+      -- Cleanup so next test can run.
+      vetDarOnParticipant v1DarName participant0
+      vetDarOnParticipant v1DarName participant1
+      vetDarOnParticipant v2DarName participant0
+      vetDarOnParticipant v2DarName participant1
+
+failing : (Text, Script ()) -> (Text, Script ())
```

Comment:
I would suggest changing this to broken, rather than failing. Failing seems like it could be "failing" as intended.